test/fast/rollback.test.ts

```diff
@@ -52,7 +52,11 @@ describe("Rollback", function() {
             rollup.filters.RollbackStatus(null, null, null),
             tx.blockHash
         );
-
+        const [event] = await rollup.queryFilter(
+            rollup.filters.RollbackTriggered(null, null),
```

Comment:
You should be able to remove these extraneous nulls here and in other filters, see https://github.com/thehubbleproject/hubble-contracts/pull/638#discussion_r679356273